01/Task01.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 
 import Alert from './../src/components/Alert';
 import { Row, Col, Alert as RBAlert } from 'react-bootstrap';
+import { ThemeProvider } from 'styled-components';
+import themeSettings from '../src/components/Alert/theme'
```

Comment:
Jeszcze lepszym rozwiązaniem jest mieć 1 motyw dla całej aplikacji i przekazywać ją w głównym komponencie np. App.js - teraz bym nie zmieniał, ale w projekcie bym to uwzględnił :)